app/controllers/api/v1/npq_participants_controller.rb

```diff
@@ -6,24 +6,28 @@ class NPQParticipantsController < Api::ApiController
       include ApiTokenAuthenticatable
       include ApiPagination
       include ApiFilter
+      include Api::ParticipantActions
 
       def index
-        npq_participant_hash = NPQParticipantSerializer.new(paginate(npq_participants)).serializable_hash
-        render json: npq_participant_hash.to_json
+        render json: NPQParticipantSerializer.new(paginate(npq_profiles)).serializable_hash.to_json
       end
 
     private
 
+      def serialized_response(profile)
+        NPQParticipantSerializer
+          .new(profile.validation_data)
+          .serializable_hash.to_json
+      end
+
       def npq_lead_provider
         current_api_token.cpd_lead_provider.npq_lead_provider
       end
 
-      def npq_participants
-        # TODO: filter out on approved participants
-        npq_participants = npq_lead_provider.npq_participants
-        npq_participants = npq_participants.where("npq_profiles.updated_at > ?", updated_since) if updated_since.present?
-        npq_participants.order(:created_at)
-        npq_participants
+      def npq_profiles
+        npq_profiles = npq_lead_provider.npq_profiles
+        npq_profiles = npq_profiles.where("updated_at > ?", updated_since) if updated_since.present?
```

Comment:
Does this work? If a participant email is changed, will this be updated?

Reply:
@ethanmills @blackrat @kicferk1:  This is good question, my visibility of where `npc_profiles` records my get updated is very limited. 
1. it's not obvious in the API were would a participant's email be updated? 
2. What other actions might be performed on the this record that would/should change a field that may not update the `updated_at` field